kubeflow/trainer/backends/kubernetes/backend.py

```
@@ -0,0 +1,580 @@
+        # Generate unique name for the TrainJob.
+        # TODO (andreyvelich): Discuss this TrainJob name generation.
+        train_job_name = random.choice(string.ascii_lowercase) + uuid.uuid4().hex[:11]
+
+        # Build the Trainer.
+        trainer_crd = models.TrainerV1alpha1Trainer()
+
+        if trainer:
+            # If users choose to use a custom training function.
+            if isinstance(trainer, types.CustomTrainer):
+                if runtime.trainer.trainer_type != types.TrainerType.CUSTOM_TRAINER:
+                    raise ValueError(f"CustomTrainer can't be used with {runtime} runtime")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)